src/intTest/resources/outbound_uat_data/acceptance/4_2.type4-immigrant-mandatory.fhir.json

```diff
@@ -0,0 +1,77 @@
+{
+  "parameter": [
+    {
+      "name": "gpTradingPartnerCode",
+      "valueString": "TES5"
+    },
+    {
+      "name": "acceptanceCode",
+      "valueString": "A"
+    },
+    {
+      "name": "acceptanceType",
+      "valueString": "4"
+    },
+    {
+      "name": "acceptanceDate",
+      "valueString": "1992-01-15"
+    },
+    {
+      "name": "dateOfUkEntry",
+      "valueString": "1991-08-06"
+    },
+    {
+      "name": "dateOfUkExit",
+      "valueString": "1968-03-05"
```

Comment:
Although we don't validate this, I think Exit date should be after Entry one - in terms of 1991>1968